test/NavDropdownSpec.tsx

```diff
@@ -44,56 +56,48 @@ describe('<NavDropdown>', () => {
       </Nav>,
     );
 
-    wrapper
-      .assertSingle('a.active')
-      .text()
-      .should.equal('DropdownItem 2 content');
+    document
```

Comment:
I think we can put a `data-testid` on DropdownItem 2, then check for the active class and text content instead of using querySelector